src/common/storage/collection_and_field_names.py

```diff
@@ -48,10 +48,10 @@
 
 ## Data product collections
 
-### Taxa frequency
+### Taxa counts
```

Comment:
The data product doesn't actually have frequency info, counts is the accurate terminology

Reply:
👍